source/_docs/installation.markdown

```diff
@@ -42,18 +42,18 @@ NUC i7/i9 | Pure power, you should not have *any* performance issues
 
 ## Recommended
 
-These install options are fully supported by Home Assistant's documentation. For example, if an integration requires that you install something to make it work on one of these methods then the integration page will document the steps required.
+Using the Home Assistant OS method is the fully supported installation option by Home Assistant's documentation. For example, if an integration requires that you install something to make it work then the integration page will document the steps required. The Home Assistant Container method, on the other hand, may have limited documentation available, for example if you want to install and use add-ons.
```

Comment:
This is an incorrect change. It is not limited to Home Assistant OS.
Also, the mention of the Container method having limited documentation is incorrect IMHO. Especially if you'd consider that the OS installation method runs the exact same containers.

Reply:
Yeah, this was the change I was most unsure about. In that case, let's skip this change. But I really do feel that it's very hard indeed to find documentation that describes for example how to install add-ons manually unless the user have installed using the HA OS method. Perhaps I have simply failed to find that documentation.

Do you understand what I'm after, and do you know of any such documentation that we can perhaps refer to? Maybe we can keep the text as is, but add a brief note that informs about the different approaches when it comes to installing add-ons?

Reply:
>  documentation that describes for example how to install add-ons manually

That is the whole point, there are no add-ons. If you want to install additional software, it is not up to us to document that. If you want to install X, please go to the website of X on how to install it.

Reply:
But it seems like Home Assistant (sometimes?) integrates with these additional software. Of course I can install e.g. NodeRed next to Home Assistant, but I am under the impression that Home Assistant would be able to integrate the NodeRed UI into Home Assistant if I had installed it as an add-on.

I agree that it's not up to you to describe how to install other software, but there has to be something going on that allows one version of Home Assistant to support add-ons while the other version does not. And I would expect that to be documented, but I cannot find that. At least apart from "this installation method magically has support for add-ons, but this similar method does not".